lettuce/cli.py

```diff
@@ -79,6 +81,44 @@ def benchmark(ctx, steps, resolution, profile_out):
         steps, str(dtype).replace("torch.float",""), mlups))
     return 0
 
+@main.command()
+@click.option("-s", "--steps", type=int, default=2001, help="Number of time steps.")
+@click.option("-r", "--resolution", type=int, default=200, help="Grid Resolution")
+@click.option("-o", "--profile-out", type=str, default="",
+              help="File to write profiling information to (default=""; no profiling information gets written).")
+@click.pass_context  # pass parameters to sub-commands
+def channelflow(ctx, steps, resolution, profile_out):
```

Comment:
Why not add the flow as a `click.option` to the convergence and benchmark functions? That avoids duplicate code.
You could add a dictionary `flow_by_name` to `lettuce.flows` (e.g.)
```
flow_by_name = {
    "TGV2D": TaylorGreenVortex2D,
    "...": ....
}
```
and then do

```
...
@click.option("--flow", type=click.Choice(flow_by_name.keys()), default="TGV2D")
...
def benchmark(...):
    flow_class = flow_by_name(flow)
    flow = flow_class(resolution=..., ...)
```